Bluepill-runner/Bluepill-runner/BPApp.m

```diff
@@ -29,6 +29,15 @@ + (instancetype)BPAppWithAppBundlePath:(NSString *)path withExtraTestBundles:(NS
     app.path = path;
     // read the files inside the Plugins directory
     NSString *xcTestsPath = [path stringByAppendingPathComponent:@"Plugins"];
+    if (!([[NSFileManager defaultManager] fileExistsAtPath:xcTestsPath isDirectory:&isdir]) && isdir) {
```

Comment:
let's add `NSFileManager *fm = [NSFileManager defaultManager];` and use that instead of calling this several times across this file